models/management.cattle.io.cluster.js

```diff
@@ -294,6 +294,10 @@ export default {
     };
   },
 
+  provisioningCluster() {
+    return this.$getters['all'](CAPI.RANCHER_CLUSTER).find(c => c.name === this.id);
```

Comment:
@vincent99 what I need to know is if this line will always return what we expect it to. This method was taken from [management.cattle.io.node](https://github.com/rancher/dashboard/blob/43d109b2bdc2672ccc9736d9c7cff74cdb7c6314/models/management.cattle.io.node.js#L104-L106). What we found was that the `id` for `management.cattle.io.cluster` isn't namespaced, so we're not sure if this is a safe way to perform our lookups.